openassessment/xblock/studio_mixin.py

```diff
@@ -183,7 +184,8 @@ def editor_context(self):
             'selected_teamset_id': self.selected_teamset_id,
             'show_rubric_during_response': self.show_rubric_during_response,
             'rubric_reuse_enabled': self.is_rubric_reuse_enabled,
-            'other_openassessments_in_course': other_openassessments_in_course,
+            'other_openassessments_in_course': json.dumps(rubric_reuse_data),
```

Comment:
If changing this variable name (which makes sense), why not also change key?

Reply:
makes sense